pixi.toml

```diff
@@ -11,22 +11,21 @@ build_sdist = "pixi run python -m build --sdist"
 
 [dependencies]
 python = ">=3.10"
-build = "*"
-rattler-build = "*"
-conda-build = "*"
-"ruamel.yaml" = "*"
+build = ">=0.7.0,<0.8"
+rattler-build = ">=0.18.1,<1"
+conda-build = ">=24.3.0,<25.0"
```

Comment:
wdyt about using this version selector?

`conda-build = "~=24.3"`

Reply:
https://github.com/prefix-dev/pixi/issues/285